src/crossfade/crossfade.cc

```diff
@@ -21,6 +21,7 @@
 #include <libaudcore/plugin.h>
 #include <libaudcore/preferences.h>
 #include <libaudcore/runtime.h>
+#include <bits/stdc++.h>
```

Comment:
I think this header is non-standard and is causing the Mac OS build to fail. Is it needed, and is there a standard C++ header you could use instead?

Reply:
So this is showing my "greenness" with C/C++ hahaha.  I foolishly included that header just to get ```exp()``` from ```<cmath>```.  Sorry about that.  

After making that swap things still compile here and functionality appears to be identical.  My environment is GCC on ArchLinux (I was going to attempt a Windows cross compile at some point).